c2corg_ui/static/js/editing/documentediting.js

```diff
@@ -624,6 +624,11 @@ app.DocumentEditingController.prototype.confirmSave = function(isValid) {
     this.alerts.addError('Form is not valid');
     return;
   }
+  
+  const data = this.scope[this.modelName];
```

Comment:
if this const is used only for the function below, I would not store it in a local variable.